commerce_coordinator/apps/ecommerce/urls.py

```diff
@@ -0,0 +1,13 @@
+"""
+ecommerce app URLS
+"""
+
+from django.urls import include, path
+
+from commerce_coordinator.apps.ecommerce.views import RedeemEnrollmentCodeView
+
+app_name = 'ecommerce'
```

Comment:
Should this match the name in `apps.py` (aka `commerce_coordinator.apps.ecommerce`)

Reply:
I see in `urls.py` that there's `ecommerce' and under that, do we have `ecommerce`, `frontend_app_ecommerce` and `titan`?